tests/typescript/classes/break.ts

```diff
@@ -33,3 +33,21 @@ export class VisTimelineComponent
 export class VisTimelineComponent2
 	implements AfterViewInit, OnChanges, OnDestroy, AndSomethingReallyReallyLong {
 }
+
+class loooooooooooooooooooong extends looooooooooooooooooong implements loooooooooooooooooooong {
```

Comment:
Do we want to fix `extends` here, or only `implements`?

Reply:
What does "to fix `extends`" mean?

Reply:
```js
class Foo extends fun(fun()) {}
```

We can have a lot of nested `fun()`, but maybe out of scope this PR

Reply:
I think that is out of scope too.

Reply:
But let's add tests, just for future, maybe we need to open an issue